lib/slax_web/websockets/issue.ex

```diff
@@ -11,24 +12,28 @@ defmodule SlaxWeb.Issue do
   def handle_event(%{"bot_id" => bot_id}) when not is_nil(bot_id), do: nil
 
   def handle_event(%{"thread_ts" => ts, "channel" => channel, "text" => text, "type" => "message"}) do
-    issues_scan = scan_text_for_issue(text)
-    prs_scan = scan_text_for_pr(text)
+    unless !is_nil(Channels.get_by_channel_id(channel)) and Channels.disabled?(channel) do
```

Comment:
Is this going to make two different DB calls to evaluate the conditional?

Reply:
I think we can handle non-existent channels and avoid the double query by doing something like
```
Channels.get_by_channel_id(channel)
if is_nil(channel) or !channel.disabled do
end
```
(doesn't use a special `disabled?()` function b/c I'm not sure it's necessary, but if there _is_ such a function IMO it should take a channel and not do a sneaky query under the hood)